internal/config/main.go

```diff
@@ -0,0 +1,105 @@
+package config
```

Comment:
What do you think of `pkg/internal/config` as the path instead?

Reply:
Why put this in pkg?  I think there will be no value for someone external to pull this into his code

Reply:
I was thinking so we concentrate where the go-code lives. Main directory (`*.go`) and one sub-directory, but it is fine - I looked at what Prometheus was doing and they have it spread out so can't be that bad.

Reply:
I have followed mostly the standard [Go structure](https://github.com/golang-standards/project-layout)

But I have skipped the CMD part as I think we should not change that much in a single step, to prevent unexpected problems